flyteidl/protos/flyteidl/admin/agent.proto

```diff
@@ -78,7 +78,7 @@ message GetTaskResponse {
 }
 
 message Resource {
-  // The state of the execution is used to control its visibility in the UI/CLI.
+  // DEPRECATED. The state of the execution is used to control its visibility in the UI/CLI.
```

Comment:
There is a `deprecated` keyword to use: https://protobuf.dev/programming-guides/proto3/#options

```
State state = 1 [deprecated = true];
```

Reply:
Thank you so much! I learned a lot

Reply:
Fixed! Thank you again!